tests/TestCase.php

```diff
@@ -92,6 +94,66 @@ protected function hasTmp(): bool
 		return defined(static::class . '::TMP');
 	}
 
+	/**
+	 * Set up a new multi language app instance with
+	 * English and German pre-installed and a test page called
+	 * `a-page` with an `article` template
+	 */
+	public function setUpMultiLanguage(): void
+	{
+		$this->app = new App([
+			'languages' => [
+				[
+					'code'    => 'en',
+					'default' => true
+				],
+				[
+					'code' => 'de'
+				]
+			],
+			'roots' => [
+				'index' => static::TMP
+			],
+			'site' => [
+				'children' => [
+					[
+						'slug'     => 'a-page',
+						'template' => 'article',
+					]
+				]
+			]
+		]);
+
+		$this->model = $this->app->page('a-page');
```

Comment:
`Concern:` I think this model part could make it hard to reuse in other tests that need e.g. more than one page in their setup.

Reply:
You are right. I've removed it from the global TestCase class and made the setUp methods more universally useful. I think we can add more arguments for other options like blueprints or whatever we need. But I would do this step by step.